src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -7586,7 +7586,8 @@ we will have to use something else.
                 }
                 Vec<ASR::expr_t*> args_; args_.reserve(al, x.n_args);
                 visit_expr_list(x.m_args, x.n_args, args_);
-                if (ASRUtils::is_array(ASRUtils::expr_type(args_[0])) &&
+                
+                if (x.n_args && ASRUtils::is_array(ASRUtils::expr_type(args_[0])) &&
```

Comment:
We can make it more explicit.
```suggestion
                if (x.n_args > 0 && ASRUtils::is_array(ASRUtils::expr_type(args_[0])) &&
```